### ISSUE TITLE
Game crazhing - fatal Java Runtime

### ISSUE THREAD

**Issue by Gumptionless** [AllTheMods/ATM-8#923]
Games been running fine for a couple of weeks, but ever since last night it will run for a couple of minutes before crashing,
Crash log attached,
[hs_err_pid9640.log](https://github.com/AllTheMods/ATM-8/files/10784290/hs_err_pid9640.log)

I have updated my drivers, updated java, it happens consistently even if i dont move on entering the world.

**Comment by Edubgr** [comment]
I have the same problem. Last time I played it was last night, haven't left the world since I updated to 1.0.11. Today when opening the world, a quick image of my world appears and crashes.
[hs_err_pid22908.log](https://github.com/AllTheMods/ATM-8/files/10785619/hs_err_pid22908.log)

**Comment by Meril502** [comment]
Exact same crash as well. (Exact same crash log as Edubgr.)

**Comment by Gumptionless** [comment]
im seeing alot on mini patches coming out on the update log, so hopefuly given a few hours it will work

**Comment by Gumptionless** [comment]
posted on reddit aswell, someone suggested disabling rubidium extras and dynamic lights,

Sadly didnt work for me, but worth a try for anyone else with this issue

**Comment by ElusiveEmu** [comment]
Same issue on my world, same log. Made a new world and it was fine until I discovered a village then it crashed. Past threads have said that it was a computer craft config issue but doing the fix in those threads did not help(changing the renderer from BEST to TBO).

**Comment by Gumptionless** [comment]
i rolled back my save, worked fine for about two hours, redid everything i lost, and crashed again

**Comment by icha-senpai** [comment]
same log same issue

**Comment by icha-senpai** [comment]
i managed to finally get a crash log 
https://pastebin.com/yaGN5y3g

its something to do with the model fix mod

**Comment by HoodiniMac** [comment]
I just play for a bit and all that progress is gone its actually frustrating

**Comment by icha-senpai** [comment]
googling this--------Problematic frame:  C  [nvoglv64.dll+0x1855e6e]

lead me to this -------https://www.reddit.com/r/fabricmc/comments/zpfwl8/fabric_crashing/

which in return made me disable rubidium which is a fork of sodium to forge and finally no more crashes

**Comment by icha-senpai** [comment]
Note: If you are uploading hs_err_pid****.log files, please make sure to redact sensitive information from the log file. Notably, the command line arguments included in the log will contain a temporary access token for your account, which could allow people to login to other Minecraft servers with your account for a short while.

**Comment by Gumptionless** [comment]
good catch, going to try removing rubidium and rubid extras, see if it works, 
plan B is roll back my driver and hope it works

**Comment by Gumptionless** [comment]
well that didnt work
![image](https://user-images.githubusercontent.com/125884150/220324583-7062d121-3cbc-4ac7-a935-d31acc6b4bc1.png)

**Comment by Gumptionless** [comment]
removing rubid extras didnt work

**Comment by Gumptionless** [comment]
newest update, tried loading a recent backup and got this error
![image](https://user-images.githubusercontent.com/125884150/220387705-7dd51d16-e8c8-457d-9d3d-e405784c4419.png)

so my backups are dead

**Comment by Gumptionless** [comment]
tried an older backup, got the same error,
went further back and got this
![image](https://user-images.githubusercontent.com/125884150/220388540-4fa1d6a6-228c-4a56-a10e-f212d5063411.png)

pretty sure this one was from befor i pulled a backup previously

**Comment by Meril502** [comment]
I suspect that this problem comes from the CPU architecture Hardware compatibility. With my intel I5 it wasn’t working like you, same messages. With my newest pc, intel I9, everything is smooth. Maybe it is Java that need to be contacted, they may have changed something in the coding that makes some mods using certain line of codes incompatible… Let’s hope I’m wrong, otherwise a lot of people will struggle on this issue.

Pierre-Olivier Emery
________________________________
De : Gumptionless ***@***.***>
Envoyé : Tuesday, February 21, 2023 10:32:49 AM
À : AllTheMods/ATM-8 ***@***.***>
Cc : Meril502 ***@***.***>; Comment ***@***.***>
Objet : Re: [AllTheMods/ATM-8] Game crazhing - fatal Java Runtime (Issue #923)


tried an older backup, got the same error,
went further back and got this
[image]<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F125884150%2F220388540-4fa1d6a6-228c-4a56-a10e-f212d5063411.png&data=05%7C01%7C%7Ca4f3a44732d34ac6afc408db1420e3fc%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638125903717877885%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=gbKCLy7NWaN%2FCe9U85NQV3V5ve17370%2By6ysKlMKNaE%3D&reserved=0>

pretty sure this one was from befor i pulled a backup previously

**Comment by icha-senpai** [comment]
> removing rubid extras didnt work

I disabled,
model gap fix
oculus
rubidium
rubidium dynamic lights
rubidium extras
its been 11 hours no crashing...so its either a combination of one or two of those mods but all i know for sure is model gap fix was in the crash report and googling hs_err_pid****.log i found a sodium error related to gpu drivers...

**Comment by Gumptionless** [comment]
new update on stuff i have tried,
totaly removed atm8 and reinstalled it,
started a new world so all the folders are made and sat in a village for 10 mins,  no crashes

Put old world into the new instance, immediate crash

rolled my game back to 1.10 and everything works totally fine,
guess ill just play a version behind and not be so excited to play a new version straight away

**Comment by derekpovah** [comment]
I kept seeing this in the logs:
```
java.lang.RuntimeException: Error building entry ars_elemental:armor/armor_aqua
```
Could it be [this](https://github.com/Alexthw46/Ars-Elemental/issues/22) simple? I manually updated client side Ars Elemental to 0.5.8.1 and didn't bother touching the server. I was able to handshake and join without issues. I will report back if I crash again.

**Comment by icha-senpai** [comment]
i updated to it and still crashed...i just wanna play atm 8 lol

**Comment by Gumptionless** [comment]
ive had multipul different mods pointed at by the errors so i think its a deeper issue,

best idea is rollback to 1.0.10 and play that for an extra week, i dont think to much major came out that we will miss

**Comment by derekpovah** [comment]
@Gumptionless Did you roll back in place or restore a backup? People on a server I'm managing that are unaffected by this have made a fair amount of progress while I've been troubleshooting; want to avoid making people angry if possible.

**Comment by Gumptionless** [comment]
i rolled back,
when i tried to restore backups during the damage all the backups where corrupted unless i went a couple of hours back,

but rolling back to 1.0.10 fixed everything

**Comment by Gumptionless** [comment]
heads up if you roll back, you do lose all your waypoints and map exploration

**Comment by cdepuydt** [comment]
I was running into this error as well, and none of the suggestions in this thread worked for me.  I turned down the quality to fast, the particles to none, turned off the dynamic lighting.  I rolled back my nvidia drivers to 522.25.  What for some random reason HAS appeared to work, is in the video options, under Extras, I switched Chunk Memory Allocator to Swap, and now I'm not crashing with a random render error, either on login, or after being on for a bit.  I don't have any idea what the root offender is, but that at least has me working now.  I have an nvidia 2070, if that somehow actually matters.
